Wireless tags platform

sergeymaysak opened the pull request:
**Description:**

Documentation for Wireless Tags platform. Includes sensors, binary sensors and switches.

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

sergeymaysak committed:
Docs for Wirelesstag platform.

sergeymaysak committed:
Added Notes section.

sergeymaysak committed:
Corrected list of push notifications events.

sergeymaysak committed:
Removed malformed notes.

sergeymaysak committed:
Removed nl.

sergeymaysak committed:
Added icon.

sergeymaysak committed:
Conformed docs to standards.

dale3h commented on source/_components/switch.wirelesstag.markdown:
```diff
@@ -0,0 +1,44 @@
+* (`light`): Control monitoring of light changes.
+* (`moisture`): Control monitoring of water level/soil moisture for water sensor.
+
+Arm/Disam of motion switch is required to receive motion and door binary sensors events.
```
- Typo in "Disarm"

dale3h commented on source/_components/sensor.wirelesstag.markdown:
```diff
@@ -0,0 +1,40 @@
+
+The following metrics can be monitored:
+
+* (`temperature`): Value is in Celcius or in Fahrenheit (accourding to your settings at Tag Manager).
```
- Typo in "Celsius"

dale3h reviewed (changes requested):
A few typos, but other than that it looks good. Now let's see if we can figure out why it's failing.

sergeymaysak committed:
Corrected typos reported during PR review.

dale3h committed:
Fixed one last typo

dale3h committed:
Small change to force rebuild

frenck commented on source/_components/binary_sensor.wirelesstag.markdown:
```diff
@@ -0,0 +1,47 @@
+logo: wirelesstag.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Push"
+---
```
`ha_version` is missing.

frenck commented on source/_components/sensor.wirelesstag.markdown:
```diff
@@ -0,0 +1,40 @@
+logo: wirelesstag.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
+---
```
`ha_version` is missing.

frenck commented on source/_components/switch.wirelesstag.markdown:
```diff
@@ -0,0 +1,44 @@
+logo: wirelesstag.png
+ha_category: Switch
+ha_iot_class: "Local Push"
+---
```
`ha_version` is missing.

sergeymaysak replied:
@frenck thanks! what version would you like me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...

frenck replied:
General recommendation is to pick the next "major"  release, which is `0.67`.

sergeymaysak replied:
@frenck got it, thx! in progress with it...

frenck commented on source/_components/wirelesstag.markdown:
```diff
@@ -0,0 +1,37 @@
+logo: wirelesstag.png
+ha_category: Hub
+ha_iot_class: "Local Push"
+---
```
`ha_version` is missing.

sergeymaysak replied:
@frenck thanks!  same here - what version would you like me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...

frenck commented on source/_components/binary_sensor.wirelesstag.markdown:
```diff
@@ -0,0 +1,47 @@
+The following conditions can be monitored:
+
+* (`presence`): On means in range, Off means out of range.
+* (`motion`): On when when movement was detected, Off when clear.
```
`On when when movement was detected` -> `On when a movement was detected`

sergeymaysak replied:
@frenck thanks! will correct shortly...

frenck commented on source/_components/binary_sensor.wirelesstag.markdown:
```diff
@@ -0,0 +1,47 @@
+
+* (`presence`): On means in range, Off means out of range.
+* (`motion`): On when when movement was detected, Off when clear.
+* (`door`): On when when door is open, Off when door is closed.
```
`On when when door is open, Off when door is closed.` -> `On when a door is open, Off when the door is closed.`

frenck commented on source/_components/switch.wirelesstag.markdown:
```diff
@@ -0,0 +1,44 @@
+* (`moisture`): Control monitoring of water level/soil moisture for water sensor.
+
+Arm/Disarm of motion switch is required to receive motion and door binary sensors events.
+Others are needed only if you want to receive push notifications from tags on specific range of changes for temperature, humidity, light or moisture.
```
- `on specific range`  -> `on a specific range`
- `changes for temperature`  -> `changes in temperature`

frenck commented:
Thanks, @sergeymaysak.
Pretty large and great contribution 🏆 

I've reviewed the docs and found small things. Could you please take a look? 👍

sergeymaysak committed:
Added ha_release: 0.67 and note how to make sure local push notifications are enabled.

sergeymaysak commented:
@frenck I've just committed requested changes and general note on main WirelessTags hub page regarding local push notification. Please have a look. Thank you )

frenck committed:
:pencil2: Tweaks

frenck committed:
:pencil2: Tweaks

frenck committed:
:pencil2: Tweaks

frenck committed:
:pencil2: Tweaks

frenck commented:
@sergeymaysak, Approved ✅ 
Made some more minor tweaks, I hope you don't mind.


Can be merged as soon as the parent PR gets merged. 😃

sergeymaysak committed:
Noted presence of Cloud Polling, added warning on https.

sergeymaysak commented:
@frenck, sorry for bothering, but based on testing of my PR by volunteers, it has been found that Tag Manager does not support `https`, so I've added `warning` on this. Additionally, I've added notion of Cloud Polling usage for each component. May I ask you to review/approve these changes?

sergeymaysak committed:
Corrected name of Tags Manager.

frenck committed:
:arrow_up: ha_release -> 0.68

frenck commented:
Merged and tagged for milestone `0.72`, since this version has already been tagged `rc`.

Thanks, @sergeymaysak! Great first-time contribution m8!

![bitmoji](https://render.bitstrips.com/v2/cpanel/359644fb-c5df-4412-9bc7-ae838696ef14-82100c49-8c0a-4ae4-9b04-3a6d3eac9cc2-v1.png?transparent=1&palette=1&width=246)
